gluten-core/src/main/scala/io/glutenproject/backendsapi/BackendsApiManager.scala

```diff
@@ -20,7 +20,7 @@ import io.glutenproject.GlutenPlugin
 
 import java.util.ServiceLoader
 
-import scala.collection.JavaConverters
+import scala.jdk.CollectionConverters._
```

Comment:
IIRC, `scala.jdk.CollectionConverters` also Scala 2.13 only

Reply:
Ah good point, looks like there is a backwards compact lib ( https://stackoverflow.com/questions/59197883/version-agnostic-way-to-convert-from-java-to-scala-collections-and-back ) I'll try for Scala 2.12 support